moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_distance_field_types.h

```diff
@@ -43,6 +43,7 @@
 #include <sstream>
 #include <memory>
 #include <float.h>
+#define protected public
```

Comment:
@tsijs These changes are not safe and can lead to undefined behavior. If you want to expose certain protected fields please add getter/setter functions.

Reply:
I know, I didnt mean to push this commit to the pull request:$